src/main/java/transforms/SplitTransform.java

```diff
@@ -28,40 +28,49 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.openlattice.shuttle.transformations.Transformation;
 import com.openlattice.shuttle.util.Constants;
+import org.apache.commons.lang3.StringUtils;
 
 import java.util.List;
 import java.util.Objects;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-@JsonIgnoreProperties(value = {TRANSFORM} )
-
-
-public class SplitTransform extends Transformation<String>  {
+public class SplitTransform extends Transformation<String> {
 
     private final String separator;
-    private final int index;
+    private final String valueelse;
+    private final int    index;
 
     /**
      * Represents a transformation to split a string
+     *
      * @param separator: separate by what?
-     * @param index: index in separated list (starts at 0!)
+     * @param index:     index in separated list (starts at 0!)
```

Comment:
This is a little confusing... can you include an example of a use case for this? Is the index supposed to be the max number of occurrences of the separator to allow? But if there's more occurrences than the max, only return one of them?